### DISCUSSION TITLE
"test connection" button or "save" button

### DISCUSSION THREAD

**Discussion by dokinlab** [creepymonster/GlucoseDirect#534] [General]
Hi! First of all i want to thx about this app! So good to use it! Thank you. 

I have a trouble this my NightScout server. I do it myself, without professional skills in dev-ops. And cannot understand it's status. 
In your's app i see setting "upload to NS". I set it. But next i dont understand what is happend, because nothing happend.

Maybe it would be good to make button "save" or another button "test connection". I think it would be easier to understand status of connection.

**Comment by philipp-1337** [comment]
@dokinlab Recently, I also set up a NightScout server and then added the URL and API Secret into Glucose Direct. It did work without any issues. I agree the user feedback isn't great, but you should see values in NightScout at least 5mins afterwards. If not, you might have an issue in your NightScout setup.

**Reply by dokinlab** [reply]
No, it was another problem. My NS server didnt have SSL, and it was working only by http. I setup ssl and after that uploading values starts immediately. 

My question is about GlucoseDirect, because I don’t understand: the problem was in NS server on in App. The button “test connection” can resolve this problem.
